docs/book/src/user/quick-start.md

```diff
@@ -484,6 +484,9 @@ kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o jsonpath={.
 
 Calico is used here as an example.
 
+{{#tabs name:"tab-deploy-cni" tabs:"AWS,Azure,Docker,GCP,vSphere,OpenStack,Metal3"}}
+{{#tab AWS|Docker|GCP|vSphere|OpenStack|Metal3}}
```

Comment:
Not sure if these instructions apply to all the other providers, let me know if some of them shouldn't be listed here

Reply:
Does it make sense to add a line about why Azure requires something different than the standard Calico? Not sure if its needed

Reply:
done